src/Button/MeasureButton/MeasureButton.example.jsx

```diff
@@ -48,6 +48,7 @@ render(
       </MeasureButton>
 
       <MeasureButton
```

Comment:
As `MeasureButton` instances are all toggle buttons actually, we could possibly wrap them into a `ToggleGroup` to enhance the example usability.

Reply:
Sadly the `ToggleGroup` doesn't work with Composed `ToggleButtons`... Maybe the context approach was better...